testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -120,6 +121,51 @@ def test_write_coordinates(self, u, tmpdir):
         with tmpdir.as_cwd():
             u.atoms.write("test.xtc")
 
+    @pytest.mark.parametrize('frames', (
+        [4],
+        [2, 3, 3, 1],
+        slice(2, 6, 1),
+    ))
+    def test_write_frames(self, u, tmpdir, frames):
+        destination = str(tmpdir / 'test.dcd')
+        selection = u.trajectory[frames]
+        ref_positions = np.stack([ts.positions for ts in selection])
+        u.atoms.write(destination, frames=frames)
+
+        u_new = mda.Universe(destination)
+        new_positions = np.stack([ts.positions for ts in u_new.trajectory])
+
+        assert_array_almost_equal(new_positions, ref_positions)
+
+    @pytest.mark.parametrize('frames', (
+        [4],
+        [2, 3, 3, 1],
+        slice(2, 6, 1),
+    ))
+    def test_write_frame_iterator(self, u, tmpdir, frames):
+        destination = str(tmpdir / 'test.dcd')
```

Comment:
If this does a `tmpdir.join('test.dcd')` then I consider it abuse of notation....

Reply:
This is the normal syntax for `pathlib.Path` objects.

Reply:
Sorry, I sounded more grouchy than I intended. I didnt know that this was a common idiom. I guess I can get use to operator overloading...